lib/game.py

```diff
@@ -41,6 +43,9 @@ def get_next_move(self) -> Move:
             # Split into 3 parts
             parts = message.split()
             if len(parts) != 3:
+                if len(parts) and parts[0] == "quit":
```

Comment:
Note: in the future, would be easier to review if this "quit" behavior was a separate PR, since it is a separate feature. For now, you can either leave this feature as part of the current PR, or revert the changes and open a separate PR - up to you.


Regarding this line: Suggest ignoring case & adding a descriptive comment, e.g.:
```python
# Check if the user wants to end the game
if len(parts) and parts[0].lower() == "quit":
```

Other comments regarding the "quit" feature:
- Add message to the user (e.g., at command line input prompt) notifying them that they can type `quit` to end the game
- Set the "quit" string itself as a constant (in the config, perhaps)